appengine-push/scripts/setup-google-api-client.sh

```diff
@@ -7,7 +7,7 @@ if [ -z "$1" ]; then
     exit
 fi
 
-TMP=${1}/pip-setup-$$.$RANDOM
+TMP=/tmp/pip-setup-$$.$RANDOM
```

Comment:
Can we use mktemp?
Here is an example which works well both on linux and MacOS.
http://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x